depthai_ros_driver/launch/depthai_node.launch

```diff
@@ -3,29 +3,18 @@
   <arg name="ns" default="depthai"/>
   <arg name="node_name" default="driver"/>
 
-  <arg name="queue_size" default="1"/>
-
   <arg name="camera_name" default="bw1097"/>
   <arg name="camera_param_uri" default="package://depthai_ros_driver/params/camera/"
     doc="Location of parameters is {camera_param_uri}/{camera_name}/{stream}.yaml"/>
 
+  <arg name="pipeline" default="depthai_ros_driver/MobilenetSSDPipeline"/>
+
   <arg name="cmd_file" default="$(find depthai_ros_driver)/resources/depthai.cmd"/>
   <arg name="calibration_file" default="$(find depthai_ros_driver)/resources/calibration/$(arg camera_name)/depthai.calib"/>
+
   <arg name="blob_file" default="$(find depthai_ros_driver)/resources/mobilenet_ssd.blob"/>
   <arg name="blob_file_config" default="$(find depthai_ros_driver)/resources/mobilenet-ssd.json"/>
-
-  <arg name="force_usb2" default="false"/>
-  <arg name="sync_video_meta" default="false"/>
-  <arg name="compute_bbox_depth" default="false"/>
-  <arg name="full_fov_nn" default="true"/>
-
-  <arg name="rgb_height" default="1080"/>
-  <arg name="rgb_fps" default="30"/>
-  <arg name="depth_height" default="720"/>
-  <arg name="depth_fps" default="30"/>
-  <arg name="shaves" default="14"/>
-  <arg name="cmx_slices" default="14"/>
-  <arg name="nn_engines" default="2"/>
+  <arg name="openvino_version" default="2" doc="integer corresponding to dai::OpenVINO::Version"/>
```

Comment:
fallback_openvino_version?

Reply:
added "mobilenetssd_" as the prefix, and cleaned the launch file to reduce confusion.